multimap/src/main/java/org/infinispan/multimap/api/MultimapCacheManager.java

```diff
@@ -0,0 +1,27 @@
+package org.infinispan.multimap.api;
+
+import org.infinispan.configuration.cache.Configuration;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+
+public interface MultimapCacheManager<K,V> {
+
+   /**
+    * Defines a named multimap cache's configuration by using the provided configuration
+    * If this cache was already configured either declaritively or programmatically this method will throw a
+    * {@link org.infinispan.commons.CacheConfigurationException}.
```

Comment:
I don't think that such checks are necessary, I just want to document that multimap "x" is just a different view to cache "x".